<launch>
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find pr2_description)/robots/pr2.urdf.xacro'" />
  
  <node name="visualize_islands" pkg="monolithic_pr2_planner_node" type="visualizeIslands" output="screen">
      <param name="island_file_name" value="$(find
      monolithic_pr2_planner_node)/experiments/islands_arm.points" />
      <param name="num_islands" value="6"  type="int" />
  </node>

  <node pkg="tf" type="static_transform_publisher" name="map_to_base_broadcaster" args="0 0 0 0 0 0 map base_link 100"/>

</launch>
